Practice2/README.md

```diff
@@ -34,9 +34,18 @@
     * images/
   * styles/
     * base/
+      * all.css
       * base.css
       * reset.css
     * components/
+      * common/
+        * all.css
+        * button.css
+        * heading-default.css
+        * heading-medium.css
+        * heading.css
```

Comment:
`heading`, `heading-default`, and `heading-medium` is just heading, so put them into one file `heading`.

Reply:
Fixed